docs-src/src/pages/design-tokens.js

```diff
@@ -170,17 +158,253 @@ function Fonts() {
+      </Table.Data>
+    </Table.Row>
+    )
+  }
+
+  return rows
+
+}
+
+function BorderRadius() {
+  return (
+    <React.Fragment>
+      <h2>Border Radius</h2>
+      <Table>
+        <Table.Head>
+          <Table.Row>
+            <Table.Header>Token</Table.Header>
+            <Table.Header>Value</Table.Header>
+          </Table.Row>
+        </Table.Head>
+        <Table.Body>{BorderRadiusRows()}</Table.Body>
+      </Table>
+    </React.Fragment>
+  )
+}
+
+// TODO is exampleStyle necessary here?  How can I eliminate this useless thang
+function ZIndexRows() {
+  
+  const rows = []
+  const zIndices = DesignTokens["z-index"]
+
+  for (const index in zIndices) {
+    const exampleStyle = {
+      zIndex: zIndices[index].value
+    };
+
+    rows.push (
+      <Table.Row>
+        <Table.Data>
+          <Table.HeaderInline>Token:</Table.HeaderInline>
+            <pre>${zIndices[index].name}</pre>
+            <p><i>{zIndices[index].comment}</i></p>
+        </Table.Data>
+        <Table.Data>
+          <Table.HeaderInline>Value:</Table.HeaderInline>
+          <pre>${zIndices[index].value}</pre>
+        </Table.Data>
+      </Table.Row>
+      )
+    }
+
+    return rows
+}
+
+function ZIndex() {
+  return (
+    <React.Fragment>
+      <h2>Z-Index</h2>
+      <Table>
+        <Table.Head>
+          <Table.Row>
+            <Table.Header>Token</Table.Header>
+            <Table.Header>Value</Table.Header>
+          </Table.Row>
+        </Table.Head>
+        <Table.Body>{ZIndexRows()}</Table.Body>
+      </Table>
+    </React.Fragment>
+  )
+}
+
+
+// TODO fix name not being displayed at same spacing as value!
+function SpacingRows() {
+
+  const rows = []
+  const spaces = DesignTokens.spacing
+
+  for (const space in spaces) {
+    const exampleStyle = {
+      spacing: spaces[space].value,
+      height: spaces[space].value,
+      width: spaces[space].value,
+    };
+
+    rows.push (
+      <Table.Row>
+        <Table.Data>
+          <Table.HeaderInline>Token:</Table.HeaderInline>
+          <pre>${spaces[space].name}</pre>
+          <p><i>{spaces[space].comment}</i></p>
+        </Table.Data>
+        <Table.Data>
+          <Table.HeaderInline>Value:</Table.HeaderInline>
+          <div style={exampleStyle}></div>
+          <pre>${spaces[space].value}</pre>
+        </Table.Data>
+      </Table.Row>
+    )
+  }
+
+  return rows
+}
+
+function Spacing() {
+  return (
+    <React.Fragment>
+      <h2>Spacing</h2>
+      <Table>
+        <Table.Head>
+          <Table.Row>
+            <Table.Header>Token</Table.Header>
+            <Table.Header>Value</Table.Header>
+          </Table.Row>
+        </Table.Head>
+        <Table.Body>{SpacingRows()}</Table.Body>
+      </Table>
+    </React.Fragment>
+  )
+}
+
+function ShadowRows() {
+  
+  const rows = []
+  const shadows = DesignTokens.box.shadow
+
+  for (const shadow in shadows) {
+    const exampleStyle = {
+      shadowing: shadows[shadow].value,
```

Comment:
This line doesn't do anything so you can remove it

Reply:
Done